tracpro/templates/polls/poll_read.html

```diff
@@ -45,10 +45,14 @@
   {% for question in questions %}
     <h3>
       {{ forloop.counter }}. {{ question.name }}:
-      <span id="question-{{ question.pk }}-heading-value-type">{{ value_type_selected|title }}</span>
+      {% if question.question_type == question.TYPE_NUMERIC %}
+        <span id="question-{{ question.pk }}-heading-value-type">{{ value_type_selected|title }}</span>
+      {% elif question.question_type == question.TYPE_MULTIPLE_CHOICE %}
+        <span>Categorical</span>
+      {% endif %}
```

Comment:
Having no title looks fine.  However, is there something that would make sense for open-ended questions?